pkgs/repo_query/lib/src/common.dart

```diff
@@ -43,6 +44,10 @@ String iso8601String(DateTime date) {
   return date.toIso8601String().substring(0, 10);
 }
 
+String overflow(String str, [int overflows = 40]) {
+  return str.length <= overflows ? str : '${str.substring(0, overflows)}...';
```

Comment:
```suggestion
  return str.length <= overflows ? str : '${str.substring(0, overflows - 3)}...';
```

To account for the three periods? :)